src/database/schema/ChatSchema.ts

```diff
@@ -101,9 +101,8 @@ class ChatSchema implements BaseDbSchema {
 
   saveIfNotExist = async (db: SQLiteDatabase) => {
     try {
-      const existingChat = await ChatSchema.getByid(db, this.id);
+      const existingChat = await ChatSchema.isExists(db, this.id);
       if (!existingChat) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `saveIfNotExist` function has been updated to use the new `isExists` method instead of the old `getById` method. This change is more efficient because it only checks for the existence of a chat, rather than retrieving the entire chat object. However, please ensure that the `isExists` method returns a boolean value and not a `ChatSchema` object.
```diff
- static async isExists(db: SQLiteDatabase, id: string): Promise<ChatSchema> {
+ static async isExists(db: SQLiteDatabase, id: string): Promise<boolean> {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->